### ISSUE TITLE
I can't get promtail to listen

### ISSUE THREAD

**Issue by CRad14** [grafana/loki#1512]
I am configuring promtail to be a syslog receiver, but for some reason I cannot get it to listen on the port configured. 

Here is the config.yaml

```yaml
server:
  http_listen_port: 9080
  grpc_listen_port: 0


positions:
  filename: /tmp/positions.yaml

clients:
  - url: http://loki_addr:3100/loki/api/v1/push

scrape_configs:
  - job_name: syslog
    syslog:
      listen_address: 0.0.0.0:10514
      labels:
        job: "syslog"
    relabel_configs:
      - source_labels: ['__syslog_message_hostname']
        target_label: 'host'
```

netstat -ntlp returns
![image](https://user-images.githubusercontent.com/4015677/72286910-ba846380-360b-11ea-828d-45da91aa6cce.png)

I am not sure what I am missing, but any help would be greatly appreciated. 

Thanks!

**Comment by rfratto** [comment]
Hi, what version of Promtail are you running? Syslog support is only in the master branch right now and hasn't been released yet.

**Comment by CRad14** [comment]
I am using version 1.2.0 head branch.

So although there are some instructions for configuring syslog, it is not available yet. Is that correct? Do you know when it will be released?

**Comment by rfratto** [comment]
> So although there are some instructions for configuring syslog, it is not available yet.

The instructions are only present in the docs for the master branch, but not in the docs for the [latest release](https://github.com/grafana/loki/tree/v1.2.0/docs).

> Do you know when it will be released?

I can't give you an exact ETA, but we're planning another release soon. In the meantime, you're free to run `grafana/promtail:master` if you want to run syslog earlier!

**Comment by slim-bean** [comment]
we should be staging what's in the next release this week, we are working on a new release process at the moment.  If all goes well there should be a new official release next week or early the following.

In the meantime as @rfratto mentioned you can use either the `master` or `latest` tags, or a specific revision to pin it down `grafana/promtail:master-3a2e5d4`

**Comment by CRad14** [comment]
Great thanks guys. I am not super familiar with docker yet. How would I pass the config.yaml to docker for grafana/promtail:master?

**Comment by rfratto** [comment]
@CRad14 Mount the directory that it's in to somewhere inside of the container, something like this:

```
docker run -v /path/to/config/on/host:/mnt/config grafana/promtail:master -config.file=/mnt/config/config.yaml
```

**Comment by CRad14** [comment]
Thanks so much for everything!

**Comment by Thrinadh-Kumpatla** [comment]
@CRad14 Is this available on this image grafana/promtail:2.1.0 ?
